frontend/src/RecordingPageComp/InstrumentSelectionComp.jsx

```diff
@@ -1,25 +1,27 @@
 /* eslint-disable react/prop-types */
 
-// import { useEffect, useState } from "react";
-
 function InstrumentSelection({ midiSounds, setSelectedInstrument }) {
+  let instrumentList = [];
+  for (
+    let i = 0;
+    i < midiSounds.current.player.loader.instrumentKeys().length;
+    i++
+  ) {
+    instrumentList.push(midiSounds.current.player.loader.instrumentInfo(i));
+  }
 
-    let instrumentList = [];
-    for (
-      let i = 0;
-      i < midiSounds.current.player.loader.instrumentKeys().length;
-      i++
-    ) {
-      instrumentList.push(midiSounds.current.player.loader.instrumentInfo(i));
-    }
-
-  
+  function updateSelected(index) {
+    setSelectedInstrument(index);
+  }
```

Comment:
what is the change here? I dont quite see the logic difference. Is it only adding space?